requirements.txt

```
@@ -1 +1 @@
-aiohttp>=3.6.0,<4
+aiohttp>=3.8.0,<4
```

It does appear to work in 3.7.4
```suggestion
aiohttp>=3.7.4,<4
```